New fixes prevent building on 32 bits arches

Fedora Rawhide, Go 1.14 beta1, i686 and armv7hl:

Build fails after changes introduced in https://github.com/google/go-jsonnet/commit/03f5ad2dc03d6bfe1f5c565fd3b02ac34765a775
```
# github.com/google/go-jsonnet
_build/src/github.com/google/go-jsonnet/builtins.go:865:82: constant -9223372036854775808 overflows int
_build/src/github.com/google/go-jsonnet/builtins.go:865:82: constant 9223372036854775807 overflows int
_build/src/github.com/google/go-jsonnet/builtins.go:869:82: constant -9223372036854775808 overflows int
_build/src/github.com/google/go-jsonnet/builtins.go:869:82: constant 9223372036854775807 overflows int
```

--- a/builtins.go
+++ b/builtins.go
@@ -862,11 +862,11 @@ func liftBitwise(f func(int64, int64) int64) func(*interpreter, traceElement, va
 			return nil, err
 		}
 		if x.value < math.MinInt64 || x.value > math.MaxInt64 {
-			msg := fmt.Sprintf("Bitwise operator argument %v outside of range [%v, %v]", x.value, math.MinInt64, math.MaxInt64)
+			msg := fmt.Sprintf("Bitwise operator argument %v outside of range [%v, %v]", x.value, int64(math.MinInt64), int64(math.MaxInt64))
 			return nil, makeRuntimeError(msg, i.getCurrentStackTrace(trace))
 		}
 		if y.value < math.MinInt64 || y.value > math.MaxInt64 {
-			msg := fmt.Sprintf("Bitwise operator argument %v outside of range [%v, %v]", y.value, math.MinInt64, math.MaxInt64)
+			msg := fmt.Sprintf("Bitwise operator argument %v outside of range [%v, %v]", y.value, int64(math.MinInt64), int64(math.MaxInt64))
 			return nil, makeRuntimeError(msg, i.getCurrentStackTrace(trace))
 		}
 		return makeDoubleCheck(i, trace, float64(f(int64(x.value), int64(y.value))))

